mewarpx/examples/capacitive_discharge.py

```diff
@@ -214,11 +222,64 @@
 sim.add_diagnostic(field_diag)
 # sim.add_diagnostic(restart_dumps)
 
+##########################
+# WarpX and mewarpx initialization
+##########################
+
+mwxrun.init_run(simulation=sim)
+
+print('Set up simulation with')
+print('  dt = %.3e s' % DT)
+print('  Total time = %.3e s (%i timesteps)' % (TOTAL_TIME, max_steps))
+print('  Diag time = %.3e s (%i timesteps)' % (DIAG_INTERVAL, diag_steps))
+
+##########################
+# Add ME diagnostic
+##########################
+
+diag_base.TextDiag(diag_steps=diag_steps, preset_string='perfdebug')
+
 ##########################
 # simulation run
 ##########################
 
-#sim.write_input_file(file_name = 'input2d')
+import matplotlib.pyplot as plt
+
+#my_solver = PoissonSolverPseudo1D(nx, ny, D_CA / nx)
 
-# step the simulation to the point where output should start
-sim.step(max_steps)
+def direct_solve():
```

Comment:
Need comments on what this function is doing, or if you don't use it here remove it and the other commented-out changes.